packages/block-editor/src/components/inner-blocks/template-picker.js

```diff
@@ -0,0 +1,58 @@
+/**
```

Comment:
Is this something we expect to only be used for inner blocks? Is there a use case for the selection screen to be used for pre-configured block attributes?

Reply:
> Is this something we expect to only be used for inner blocks? Is there a use case for the selection screen to be used for pre-configured block attributes?

That's a good point. It's unclear to me how this might look from a design perspective, since the current concepts seem to align quite closely with how _blocks_ are arranged (technically aligning to the template option), though I don't see any reason why it couldn't be that the options configure a block's own attributes. It speaks a bit to the original questions about whether this be proposed an enhancement to the Placeholder component rather than to the InnerBlocks component, which could be more easily adapted. I still think we could extract something common regardless.

At least as experimental props, we could still opt toward the Placeholder improvements if own-block attributes option selection pans out to be a good idea to pursue.